crates/ruma-client-api/src/r0/session/login/fallback.rs

```diff
@@ -0,0 +1,49 @@
+//! [/_matrix/static/client/login/](https://spec.matrix.org/unstable/client-server-api/#login-fallback)
```

Comment:
The version of this API with `device_id` and such isn't in https://matrix.org/docs/spec/client_server/r0.6.0#login-fallback, so i picked the newer version.

Reply:
Hm, okay. The HTTP method should be included here though.

```suggestion
//! [GET /_matrix/static/client/login/](https://spec.matrix.org/unstable/client-server-api/#login-fallback)
```